auth.py

```diff
@@ -10,9 +12,12 @@ def get_credentials():
     return username, password
 
 def read_pwdb():
-    with open(PATH, "rt") as f:
-        pwdb = json.load(f)
-    return pwdb
+    if Path(PATH).exists():
+        with open(PATH, "rt") as f:
+            pwdb = json.load(f)
```

Comment:
Doing the existence check separately and then opening the file is not good, because it introduces a TOCTOU race. Instead, the correct way is to try to open the file, and catch the exception if that fails:
```python
try:
   … open(PATH, 'rt') …
except IOError:
   return {}
```